packages/allure-js-commons/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.json",
   "include": [
     "./src/**/*",
-    "index.ts"
+    "index.ts",
+    "../allure-cucumberjs/src/BufferEncodingUtility.ts"
```

Comment:
I'm not sure, that it's good idea to import types from dependent package. Better, to define the type or utility in the `commons` package and then, reuse it in `cucumberjs` one

Reply:
This was an accidental include where I placed the file there originally, then I moved it to the other project but it retained this reference.  I have deleted it now.